doc/release-notes/0.35/0.35.md

```diff
@@ -141,4 +141,4 @@ on x64 platforms, but builds with OpenJDK 17 and later also require more stack s
 
 ## Other changes
 
-A full commit history for this release is available at [Eclipse OpenJ9 v0.35.0](https://github.com/eclipse-openj9/openj9/releases/tag/openj9-0.35.0).
+A full commit history for this release is available at [Eclipse OpenJ9 v0.35.0](https://www.eclipse.org/openj9/docs/version0.35/).
```

Comment:
This isn't the correct link to update. It's the link under "Notable changes in this release" that should change from https://eclipse-openj9.github.io/openj9-docs/version0.35/ to https://www.eclipse.org/openj9/docs/version0.36/